identity-service/relay-rate-limit.json

```diff
@@ -0,0 +1,147 @@
+{
+  "CreateUser": {
+    "owner": 10,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "UpdateUser": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "VerifyUser": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "UpdateUserReplicaSet": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "CreateTrack": {
+    "owner": 100,
+    "app": 20,
+    "whitelist": 1000
+  },
+  "UpdateTrack": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "DeleteTrack": {
+    "owner": 100,
+    "app": 20,
+    "whitelist": 1000
+  },
+  "CreatePlaylist": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "UpdatePlaylist": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "DeletePlaylist": {
+    "owner": 100,
+    "app": 5,
+    "whitelist": 1000
+  },
+  "FollowUser": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnfollowUser": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "SubscribeUser": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnsubscribeUser": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "SaveTrack": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnsaveTrack": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "RepostTrack": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnrepostTrack": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "SavePlaylist": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnsavePlaylist": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "RepostPlaylist": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "UnrepostPlaylist": {
+    "owner": 100,
+    "app": 10,
+    "whitelist": 1000
+  },
+  "CreateNotification": {
+    "owner": 0,
+    "app": 0,
+    "whitelist": 1000
+  },
+  "ViewNotification": {
+    "owner": 100,
+    "app": 0,
+    "whitelist": 1000
+  },
+  "ViewPlaylistNotification": {
+    "owner": 100,
+    "app": 100,
+    "whitelist": 1000
+  },
+  "CreateDeveloperApp": {
```

Comment:
@isaacsolo I see this says 5 apps per owner per hour, but we still want 3 apps max total per user right? was going to add that check to the EM indexer

Reply:
Ooh ok I'll change it to 3